### DISCUSSION TITLE
Renault Talisman 2017 - Bose sound problem

### DISCUSSION THREAD

**Discussion by manssonph** [cedricp/ddt4all#1010] [Q&A]
Hi,

In my Talisman i have the first generation of hardware, so i decided to buy a used one from an another car. The new one is HW version 6.X and SW version 9.X.

Now to the problem, all the things i could think of worked just fine. But i have no sound...
I did try to change the following settings

ECU - LOUDSPEAKERS - Change Premium Amplifier to Fitted;
ECU - LOUDSPEAKERS - Change Rear tweeters to Not Fitted;
ADAS - ADAS PRESENCE - Change Sound configuration to AudioBox Ampli;

Rebooted the system by holding the power button down but still no luck.

Is there anywhere else that i might need to change settings?

**Comment by matzel82** [comment]
The new HW have to configure to your car. Change HW is evertime very special.

**Reply by manssonph** [reply]
That I understand. However what I need to know is more related to the audio. 

I saw in dd4all tool “audio_configuration” and “output_information” could this be the case that I need to alter those?

Or is there anything else that might be needed.

**Comment by manssonph** [comment]
I figured it out. The settings I changed in DEV menu didn’t stick. I got a ELM327 and changed the settings via this tool. Now everything works as expected.
